contracts/TalentLayerEscrow.sol

```diff
@@ -105,15 +105,15 @@ contract TalentLayerEscrow is Initializable, ERC2771RecipientUpgradeable, UUPSUp
 
     /**
      * @notice Emitted after each payment
-     * @param _transactionId The id of the transaction.
      * @param _paymentType Whether the payment is a release or a reimbursement.
+     * @param _transactionId The id of the transaction.
      * @param _amount The amount paid.
      * @param _token The address of the token used for the payment.
      * @param _serviceId The id of the concerned service.
      */
     event Payment(
-        uint256 _transactionId,
```

Comment:
fix changes doesn't optim gas, I think the only packing possible is to pack the enum (uint8 1 byte) with the address (20 bytes)
done here: https://github.com/TalentLayer/talentlayer-id-contracts/pull/133/files

Reply:
tbh, i think i have noticed some weird hacks on packing, i will research further, I am not sure if its a fluke or actually consistent